js_modules/dagit/packages/ui/package.json

```diff
@@ -20,10 +24,17 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-is": "^17.0.2",
+    "react-router": "^5.2.1",
+    "react-router-dom": "^5.3.0",
     "styled-components": "^5.3.3"
   },
   "dependencies": {
+    "deepmerge": "^4.2.2",
```

Comment:
It may not matter but it feels like some of these should be devDependencies? Maybe not since we're including src/ui directly in our custom CRAs, but I'd expect to see the babel and @types ones not in the official package deps?

Reply:
Regrettably, `deepmerge` a real dep. I think we should go through and clean up some of the deps we have in there right now. `react-router` is especially bugging me -- I'm going to pull out all the RR-specific stuff and put it in `core`.